pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml

```diff
@@ -52,20 +52,6 @@
 
 
   <!-- Overriding standard one to exclude log bundle which will not work without pax-logging -->
-  <feature name="management" description="Provide a JMX MBeanServer and a set of MBeans in Karaf" version="5.0">
```

Comment:
I was getting errors when I included this feature. May be I can try this again and see what happens